backend/api/src/main/kotlin/io/tolgee/hateoas/notification/NotificationPagedModel.kt

```diff
@@ -0,0 +1,21 @@
+package io.tolgee.hateoas.notification
+
+import org.springframework.hateoas.Link
+import org.springframework.hateoas.Links
+import org.springframework.hateoas.PagedModel
+
+class NotificationPagedModel(
+  content: Collection<NotificationModel> = emptyList(),
+  metadata: PageMetadata? = null,
+  links: Iterable<Link> = Links.NONE,
+  val unseenCount: Int,
```

Comment:
I don't think creating special model is the right way to do that. In sake of keeping the API consistent, I would much rather add filter for seen notifications. That way, you will get unseen count in the "total count" in the page info object.